satellite/satellitedb/database.go

```diff
@@ -71,6 +72,11 @@ func (db *DB) BandwidthAgreement() bwagreement.DB {
 // 	return &accountingDB{db: db.db}
 // }
 
+// Irreparabledb is a getter for DB repository
```

Comment:
This comment isn't informative, use something like `// IrreparableDB returns database for storing segments that failed repair`

Reply:
updated the comment